app/controllers/users_controller.rb

```diff
@@ -0,0 +1,30 @@
+class UsersController < ApplicationController
+  include UsersHelper
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+
+    if @user.save
+      redirect_to new_user_path
+    else
+      redirect_to new_user_path, alert: 'Error creating user.'
```

Comment:
Please use `render :new` when the user was not saved, instead of redirecting. The redirection will remove all prefilled data from the form whereas `render` preserves it ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_create.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms)).

Reply:
Please make the change requested above.